core/src/main/java/com/nerds/gamejam/ecs/system/PlanetMapGeneratorSystem.java

```diff
@@ -51,7 +51,7 @@ private void createMap() {
                planetCount++;
             }
             if (planetCount >= maxPlanets) {
-                //createBackgroundAnimationObjects();
+                createBackgroundAnimationObjects();
```

Comment:
Where is the actual increasing the camera speed part again? :stuck_out_tongue: Also I think Mark's done that in the changes he put to fix my AreWeScaly branch, so you probably can just take that :smile: